independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/buildfile/MavenProjectBuildFile.java

```diff
@@ -86,8 +86,8 @@ public static QuarkusProject getProject(Artifact projectPom, Model projectModel,
         final List<ArtifactCoords> importedPlatforms;
         final String quarkusVersion;
         if (projectPom == null) {
-            managedDeps = Collections.emptyList();
-            deps = () -> Collections.emptyList();
+            managedDeps = List.of();
+            deps = List::of;
```

Comment:
Out of curiosity, what is the difference between both?

Reply:
In this case, it's practically equivalent. I probably modified this class in some other way originally and then reverted but these lines stayed.